### ISSUE TITLE
Python assertIs<Type>

### ISSUE THREAD

**Issue by maximiliankaul** [Fraunhofer-AISEC/cpg#1737]
Check all tests and replace:

```kotlin
val foo = getFoo() as? SomeNodeType
assertNotNull(foo)
```

with

```kotlin
val foo = getFoo()
assertIs<SomeNodeType>(foo)
```

**Comment by oxisto** [comment]
You could even inline in if you want `val foo = assertIs<SomeNodetype>(getFoo())`

**Comment by maximiliankaul** [comment]
We should probably have some review guidelines which list these style guides.

**Comment by oxisto** [comment]
Was addressed by #1734
